lib/Command/Configure.php

```diff
@@ -109,7 +110,7 @@ private function configureSignal(InputInterface $input, OutputInterface $output)
 	private function configureSms(InputInterface $input, OutputInterface $output) {
 		$helper = $this->getHelper('question');
 
-		$providerQuestion = new Question('Please choose a SMS provider (websms, playsms, clockworksms, puzzelsms, ecallsms, voipms, huawei_e3531, spryng, sms77io, ovh, clickatellcentral, clicksend): ', 'websms');
+		$providerQuestion = new Question('Please choose a SMS provider (websms, playsms, clockworksms, plivo, puzzelsms, ecallsms, voipms, huawei_e3531, spryng, sms77io, ovh, clickatellcentral, clicksend): ', 'websms');
```

Comment:
personally like the alphabetical order, but latest updates always append to the end, so I would suggest to add it as last mention.

Reply:
I agree. I'll rearrange and put in alphabetical order

Reply:
thanks!